src/process_util.rs

```diff
@@ -135,8 +153,11 @@ pub fn run_and_wait(cmd: &mut Command)
                         Stop(..) | Continue(..) | Death(..) => { }
                     }
                 }
-                try!(tty_guard.check().map_err(|e|
-                    format!("Error handling tty: {}", e)));
+                if take_tty {
+                    try!(tty_guard.check().map_err(|e|
+                        format!("Error handling tty: {}", e)));
+
+                }
             }
             _ => unreachable!(),
```

Comment:
Agree. Should we ignore those signals or propagate `SIGTSTP` to children and send `SIGSTOP` self? (think we should stop all processes)

Now I can't reproduce trapping vagga `SIGTTIN` or `SIGTTOU` signals (only manually via `kill` command).

It is very interesting that vagga launcher received `SIGTTIN` (I rather expected `SIGTTOU`) when stdout was redirected into `less`:

```
$ ps -g 7135 -H -o pid,pgrp,sid,tty,stat,cmd
  PID  PGRP   SID TT       STAT CMD
 7135  7135  7135 pts/20   Ss   -zsh
 9495  9495  7135 pts/20   S      ../../vagga _run py2-alpine python -c  import time time.sleep(2) for i in range(1000000):     print i 
 9504  9504  7135 pts/20   S+       vagga_wrapper --root py2-alpine.202203b7 _run py2-alpine python -c  import time time.sleep(2) for i in rang
 9507  9507  7135 pts/20   S          /usr/bin/python -c  import time time.sleep(2) for i in range(1000000):     print i 
 9496  9495  7135 pts/20   S      less
```

Think this was due to taking tty by `vagga_wrapper` but still don't understand why did launcher receive `SIGTTIN`?